libminifi/include/FlowController.h

```diff
@@ -225,11 +225,11 @@ class FlowController : public core::controller::ForwardingControllerServiceProvi
   // Whether it has already been initialized (load the flow XML already)
   std::atomic<bool> initialized_;
   // Flow Timer Scheduler
```

Comment:
There are a lot of useless comments in FlowController.h and FlowController.cpp like this, could you please remove these while you are at it? :pleading_face:

Reply:
removed some comments that didn't seem very useful